src/imgui/ImGuiSymbols.cc

```diff
@@ -189,15 +263,69 @@ void ImGuiSymbols::paint(MSXMotherBoard* /*motherBoard*/)
 
 		im::TreeNode("Symbols per file", ImGuiTreeNodeFlags_DefaultOpen, [&]{
 			auto drawFile = [&](const FileInfo& info) {
+				auto* file = symbolManager.findFile(info.filename);
 				im::StyleColor(!info.error.empty(), ImGuiCol_Text, getColor(imColor::ERROR), [&]{
 					auto title = strCat("File: ", info.filename);
 					im::TreeNode(title.c_str(), [&]{
 						if (!info.error.empty()) {
 							ImGui::TextUnformatted(info.error);
 						}
 						im::StyleColor(ImGuiCol_Text, getColor(imColor::TEXT), [&]{
+							auto arrowSize = ImGui::GetFrameHeight();
+							auto extra = arrowSize + 2.0f * style.FramePadding.x;
+							ImGui::SetNextItemWidth(ImGui::CalcTextSize("3-3").x + extra);
+						        ImGui::AlignTextToFramePadding();
+
+							// build slot-subslot preview string
+							auto preview = [&]{
+								if (file->slot) {
+									int ps = *file->slot & 3;
+									int ss = (*file->slot >> 2) & 3;
+									return isSlotExpanded(motherBoard, ps) ? strCat(ps, '-', ss)
+									                                       : strCat(ps);
+								} else {
+									return std::string("-");
+								}
+							}();
```

Comment:
I didn't notice this in my review yesterday (sorry).
This `preview` helper function is almost the same as the logic that draws the slot column (around line 224).
Maybe both should use the same helper?
Note: currently both are similar but not exactly the same (different behavior when `motherBoard == nullptr`), but that difference is probably not intended (a bug?).
```
static std::string formatSlot(std::optional<uint8_t> slot, MSXMotherBoard* motherBoard)
{
    if (!slot) return "-";
    int ps = (*slot >> 0) & 3;
    int ss = (*slot >> 2) & 3;
    return isSlotExpanded(motherBoard, ps) ? strCat(ps, '-', ss)
                                           : strCat(ps);
}
```
And then calls as:
* `ImGui::TextUnformatted(formatSlot(sym.slot(symbolManager), motherBoard));` for the slot column (this immediately avoids the repeated calls to `sym.slot(...)`).
* `auto preview = formatSlot(file->slot, motherBoard);` for the combo-box preview.